app/controllers/info_controller.rb

```diff
@@ -65,7 +65,7 @@ def get_part_urls(content, slug)
   end
 
   def is_multipart(part_urls, document_type)
-    (part_urls.length != 0) || (document_type == 'smart-answer')
+    !part_urls.empty? || (document_type == 'smart-answer')
```

Comment:
Yeah, to be honest, I was just mindlessly following the linter.

I've changed the relevant commit to 99fa8d9d351585b1413c55118c5b57a34bc40b35